texar/torch/run/executor.py

```diff
@@ -714,6 +716,9 @@ def __init__(self, model: nn.Module,
                  test_data: OptionalDict[DataBase] = None,
                  batching_strategy: Optional[BatchingStrategy] = None,
                  device: Optional[torch.device] = None,
+                 # tbX logging
+                 tbx_logging_dir: Optional[str] = None,
+                 tbx_log_every: Optional[Condition] = None,
```

Comment:
The original tensorboard arguments at the bottom can safely go away now, I think.

Reply:
Also, do we support setting experiment names now?

Reply:
Currently, we don't. TBX logs are written directly to `tbx_logging_dir`. Removed tensorboard args in bbaf278